static/css/style.css

```diff
@@ -245,8 +253,40 @@ html .all-auth-container {
 .list li{
     font-size: 0.5em;
     text-align: left;
+}
+
+/* View Hackathon Page */
+
+.page-title:after {
```

Comment:
CSS3 standard is `::after` instead of `:after`.

Reply:
Changed as suggested.